### DISCUSSION TITLE
Sim racing

### DISCUSSION THREAD

**Discussion by iskandar9ll** [Whisky-App/Whisky#528] [Q&A] [answered]
Does whisky support sim racing wheels? 
I tried assetto corsa compititzione and it is running perfect, over 70 FPS, max graphics. 
(M2 Pro, 16GB)
<img width="1512" alt="Screenshot 2023-10-16 at 10 04 19" src="https://github.com/Whisky-App/Whisky/assets/148012608/81c37317-04ee-40db-8657-72a24259d591">

**Comment by RinatGumarov** [comment]
How did you run assetto? I've tried to install first version via steam but when I try to run it I only see that it wants to install directx and then it just silently stops. I've also installed .net framework and fanatec driver. But when I open it it just do nothing. However I can run other games such as Payday2

**Reply by iskandar9ll** [reply]
I didn‘t do anything special. Assetto Corsa Competitzione just installed and run at first try via steam. For the first time it takes a bit long to open the game, i also thought that it won’t run, but then the window popped out. But i couldn’t run Assetto Corsa and Raceroom, tried also other settings.

**Reply by bandoOntop** [reply]
I got the regular assetto corsa running using whisky at first I had the same problem couldn’t launch it all u gotta do is create a new bottle and before adding steam or assetto corsa download the Microsoft 4.5.2 framework inside the bottle then download the visual cc++runtime both can be found by clicking on winetricks dlls and look for dotnet452 and hit run and also look for vcrun2019 or 2022 and download it as well.
Then just install steam and assetto and u should be good 
![image](https://github.com/user-attachments/assets/d9cbe8c6-f40d-4b68-913a-8dffafeb1225)
![image](https://github.com/user-attachments/assets/bea22626-2d01-4d73-ada6-ae6bddab890d)
![image](https://github.com/user-attachments/assets/a3638c26-6d5e-44ce-a7eb-ade6273ba66c)

**Reply by iskandar9ll** [reply]
Okay, thanks for the answer. I will give it a try. Have you tried evo yet?

**Reply by bandoOntop** [reply]
No I haven’t yet tried it but I’m sure it might work ion kn maybe but one thing I’m trying to install but get no results is the content manager for the assetto

**Comment by drumcorpse** [comment, marked as answer]
I always wanted to try BeamNG. I was able to get it running and playable, but whenever I connect my Fanatec GT DD it crashes the bottle immediately. Tried installing Fanatecs drivers but still does the same thing. If I connect the wheel in the other modes it will connect, but the steering input and pedal input isn't recognized, and some or all of the buttons will work.

**Comment by GaloBianiFilms** [comment]
Ok I’m new to the whole racing wheel. I’m trying to get it recognized by my Mac mini w the M4 chip. What will I need to get it to work as best it can. I have the Thrustmaster T GT-II. Can anyone help or provide some insight or point me in the direction I should be looking?

**Comment by iskandar9ll** [comment]
I have tried thrusmaster t300 and it didn’t work. I tried lots of things.
Mac doesn’t recognize the wheel properly. And i just got tired of this shit
and stopped trying to do with my mac what it is not for.
